domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/trips/BikesAllowedStatus.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.gtfs.trips;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum BikesAllowedStatus {
```

Comment:
See how the wheelchair allowed and bike allowed concept can be implemented once and reused by different GTFS entities (instead of having them specific to one kind of entities (here, trips))

Reply:
Implementing a "generic" BikesAllowedStatus in #100